known-issues.md

```diff
@@ -259,28 +259,39 @@ Note: There is a known issue with Visual Studio 2015 Update 3 RC, as a result of
 
 ### Missing SDK
 
-If you open an RC2 project (or RTW) and the corresponding .NET Core SDK is missing you'll get a dialog box similar to the following.
+If you open a Preview 2 (1.0.1) project and the corresponding .NET Core SDK is missing you'll get a dialog box similar to the following.
```

Comment:
How about just "a .NET Core project"? The dialog will appear for any case where the corresponding .NET Core SDK is missing.